resources/default-config.kdl

```diff
@@ -15,6 +15,9 @@ input {
             // For example:
             // layout "us,ru"
             // options "grp:win_space_toggle,compose:ralt,ctrl:nocaps"
+
+            // You can also specify a file from where a keymap will be loaded.
+            // file "~/.config/keymap.xkb"
```

Comment:
```suggestion
```
I think we can leave it out of the default config, kind of a niche thing it feels.